app/models/item.rb

```diff
@@ -1,4 +1,5 @@
 class Item <ApplicationRecord
+  after_initialize :set_defaults
```

Comment:
Not sure how important this is, but this might look cleaner with different describe blocks for validations/relationships.

Reply:
My mistake. I thought I was looking at the spec file